sample-configs/sample-app.yaml

```diff
@@ -42,7 +42,7 @@ spec:
               value: http://my-collector-collector:4317
             - name: OTEL_RESOURCE_ATTRIBUTES
               value: service.namespace=GettingStarted,service.name=GettingStartedService
-          image: public.ecr.aws/aws-otel-test/aws-otel-java-test-spark:v0.11.0
+          image: public.ecr.aws/aws-otel-test/aws-otel-java-spark:1.17.0
```

Comment:
Should we instead target `latest` here? This sample app is continually updated from the `aws-otel-java-instrumentation` repo. This config is meant to be used as a starting point and pinning to a version that falls behind the collector could cause errors. 

@Aneurysm9 do you have an opinion of using `latest` in sample documentation?